replacer.go

```diff
@@ -66,6 +66,17 @@ func (r *Replacer) Get(variable string) (interface{}, bool) {
 	return nil, false
 }
 
+// GetString gets a value from the replacer. It returns
+// the value and whether the variable was known.
```

Comment:
Explain what makes this different from Get -- that it converts it to a string representation.
```suggestion
// GetString is the same as Get, but coerces the value to a string representation.
```

Reply:
This still needs attention

Reply:
Thanks for the suggested comment